style.css

```diff
@@ -0,0 +1,48 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app {    
+    background-color: #FFFFFF;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;       
+    max-width: 1440px;
+    max-height: 704px;     
+}
+
+.feature-details {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    max-width: 1110px;
+    max-height: 128px;
+    margin-left: 165px;
+    margin-top: 64px;    
+}    
+
+.feature-title {
+    font-family: 'Inter', sans-serif;
+    font-size: 40px;
+    font-weight: 800;
+    width: 540px;
+    height: 108px;      
```

Comment:
The title or a text element shouldn't have a width property. It can have a max-width in case you want it to be in multiple lines. Also, the height is not relevant here.